templates/zerver/app/home.html

```diff
@@ -124,9 +124,9 @@ <h4>{{ _("You haven't been mentioned yet!") }}</h4>
     <div id="empty_search_narrow_message" class="empty_feed_notice">
         <h4>{{ _('Nobody has talked about that yet!') }}</h4>
     </div>
-    <div class="message_table focused_table" id="zhome">
+    <div class="message_table focused_table" id="zhome" role="region" aria-live="polite" aria-label="{{ _('Messages') }}">
     </div>
-    <div class="message_table" id="zfilt">
+    <div class="message_table" id="zfilt" role="region" aria-live="polite" aria-label="{{ _('Messages') }}">
```

Comment:
How does `aria-live` interact with hidden/visible divs?  Generally the model is that one of `zfilt` and `zhome` is visible at a time, and we probably only want to read messages in the one that's visible.

Reply:
I don't know if a live region is the right solution here. For every new message, it will also read all the actions etc related to it. This might be a good intermediate solution, but i think we'll need a live region to expose alerts (such as incoming messages).

Btw, you might consider `role=log` instead of `role=region` here.